.github/workflows/OS-UnitTests.yml

```diff
@@ -0,0 +1,43 @@
+name: OS Unit Tests
+
+on:
+  push:
+    branches:
+      - staging
+      - trying
+
```

Comment:
This won't run on master?

Reply:
No, we can add it but if it passes staging isn't it redundant to re-run the job?

Reply:
> if it passes staging isn't it redundant to re-run the job?

Yes, I agree. We will hopefully add nightly or weekly test runs on master going forward.